src/messaging/tests/echo/echo_responder.cpp

```diff
@@ -120,6 +144,18 @@ int main(int argc, char * argv[])
         SuccessOrExit(err);
     }
 
+    if (!disableUDC)
+    {
+        gUDCManager.Init(chip::Transport::UdpListenParameters(&chip::DeviceLayer::InetLayer)
+                             .SetAddressType(chip::Inet::kIPAddressType_IPv4)
+                             .SetListenPort(CHIP_PORT + 3));
```

Comment:
Is there any reason not to move immediately to the use of an ephemeral port for this service?  That's our eventual destination.  And since commissioner discovery is a core component of this feature anyway, it seems like it would make sense to do this sooner than later.

Reply:
addressed in latest changes